app/controllers/holiday_entitlements_controller.rb

```diff
@@ -34,7 +34,7 @@ def create
   def update
     respond_to do |format|
       if @holiday_entitlement.update(holiday_entitlement_params)
-        format.html { redirect_to holiday_entitlements_path, notice: 'Holiday entitlement was successfully updated.' }
+        format.html { redirect_to admin_holiday_entitlements_path, notice: 'Holiday entitlement was successfully updated.' }
```

Comment:
Line is too long. [124/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.